build/toolchain-aarch64-linux-android.cmake

```diff
@@ -50,9 +50,9 @@ if(WIN32)
 endif()
 set(CMAKE_C_COMPILER   ${TOOLCHAIN}/bin/${ARCH}${API_LEVEL}-clang${COMPILER_SUFFIX})
 set(CMAKE_CXX_COMPILER ${TOOLCHAIN}/bin/${ARCH}${API_LEVEL}-clang++${COMPILER_SUFFIX})
-set(CMAKE_AR           ${TOOLCHAIN}/bin/${ARCH}-ar${TOOL_SUFFIX}  CACHE FILEPATH "Archiver")
```

Comment:
I think we still need to specify `TOOL_SUFFIX` for Windows support.

Reply:
ah right my bad